tests.py

```diff
@@ -1,107 +1,192 @@
+#!/usr/bin/python
```

Comment:
I'll look at this another day. It would be good to have some sort of unit test, e.g. some data with known segmentation results. I'll think about this.